### PR TITLE
WebSocket 2.2 plan update

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [X] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [X] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [X] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [X] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [X] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [X] **N/A** Declares optional features if there are any (Mark with **N/A** and check if none)
- [X] Includes minimum Java SE version
- [X] **N/A** Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [X] Link to updated release record

### CODE REVIEW

File: websocket/2.2/_index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+### New features, enhancements or additions
+* None
+
+### Removals, deprecations or backwards incompatible changes
+* All references to the SecurityManager have been removed
+
+### Minimum Java SE Version
+**Java SE 8 or higher**
+
+
```
Comment (kazumura) [inline websocket/2.2/_index.md:20]: Please add here
``` # Details```

**Review by kazumura — CHANGES_REQUESTED** [review]

File: websocket/2.2/_index.md (hunk 1 above)
Comment (markt-asf) [reply websocket/2.2/_index.md]: Fixed.

**Comment by kazumura** [comment]
@markt-asf   
Thanks for update.  
But a preview page creation was failed.  
I 'm not sure the reason, but one thing I noticed is you have not created the release page yet.
"https://projects.eclipse.org/projects/ee4j.websocket/releases/2.2"  
So please create the release page and rerun preview page creation.

**Comment by markt-asf** [comment]
@kazumura Release page created. I don't have karma to rerun preview page creation.

**Comment by kazumura** [comment]
Not only this PR, but any commits since last week fail to create preview pages.

**Review by kazumura — APPROVED** [review]

**Comment by kazumura** [comment]
Ballot passed.
https://www.eclipse.org/lists/jakarta.ee-spec/msg03011.html

**Pull request merged**